<launch>
    <arg name="detection_topic" default="cusub_perception/darknet_ros/bounding_boxes" />
    <arg name="image_topic" default="cusub_common/occam/image0" />
    <arg name="detection_thresh" default="0.8" />
    <arg name="reseed_thresh" default="0.9" />
    <arg name="debug_mode" default="true" />
    
    <group ns="leviathan">
        <node pkg="nodelet" type="nodelet" name="tracking_standalone_nodelet" args="manager" output="screen"/>
        <node pkg="nodelet" type="nodelet" name="tracking" args="load tracking/Tracking tracking_standalone_nodelet" output="screen">
            <param name="detection_topic" value="$(arg detection_topic)"/>
            <param name="image_topic" value="$(arg image_topic)"/>
            <param name="detection_thresh" value="$(arg detection_thresh)"/>
            <param name="reseed_thresh" value="$(arg reseed_thresh)"/>
            <param name="debug_mode" value="$(arg debug_mode)"/>
        </node>
    </group>

</launch>
